src/configs/zig.rs

```diff
@@ -1,22 +1,21 @@
-use crate::config::{ModuleConfig, RootModuleConfig, SegmentConfig};
+use crate::config::{ModuleConfig, RootModuleConfig};
 
-use ansi_term::{Color, Style};
 use starship_module_config_derive::ModuleConfig;
 
 #[derive(Clone, ModuleConfig)]
 pub struct ZigConfig<'a> {
-    pub symbol: SegmentConfig<'a>,
-    pub version: SegmentConfig<'a>,
-    pub style: Style,
+    pub format: &'a str,
+    pub symbol: &'a str,
+    pub style: &'a str,
     pub disabled: bool,
 }
 
 impl<'a> RootModuleConfig<'a> for ZigConfig<'a> {
     fn new() -> Self {
         ZigConfig {
-            symbol: SegmentConfig::new("↯ "),
-            version: SegmentConfig::default(),
-            style: Color::Yellow.bold(),
+            format: "via [$symbol$version]($style)",
```

Comment:
I think you need to add a trailing space here to match the output of the old module and the updated config docs.
This should also fix the tests.

```suggestion
            format: "via [$symbol$version]($style) ",
```